nautobot/extras/models/jobs.py

```diff
@@ -1127,6 +1129,16 @@ def schedule(self):
 
         return self.to_cron()
 
+    @property
+    def queue(self):
```

Comment:
Similarly probably need to add this field explicitly to the API serializer and GraphQL.

Reply:
Added on the API side but it looks like ScheduledJob does not support GraphQL to begin with

Reply:
Hm okay. Probably would be desirable to add that but I'm fine with considering that out-of-scope for now. (See also #5917)